presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoPreparedStatement.java

```diff
@@ -455,7 +458,26 @@ public void setArray(int parameterIndex, Array x)
     public ResultSetMetaData getMetaData()
             throws SQLException
     {
-        throw new SQLFeatureNotSupportedException("getMetaData");
+        try (PrestoStatement statement = (PrestoStatement) getConnection().createStatement()) {
```

Comment:
We should be able to use `Statement` here. I don't see anything that requires `PrestoStatement`.